app/controllers/scenarios_controller.rb

```diff
@@ -69,6 +70,7 @@ def edit
 
   def create
     @scenario = current_user.scenarios.build(params[:scenario])
```

Comment:
It seems like we'd want to write a migration and add a new column to scenarios to support an icon.